packages/mds-geography/api.ts

```diff
@@ -0,0 +1,185 @@
+import express from 'express'
+import db from '@mds-core/mds-db'
+
+import { pathsFor, ServerError, NotFoundError } from '@mds-core/mds-utils'
+import { geographyValidationDetails } from '@mds-core/mds-schema-validators'
+import log from '@mds-core/mds-logger'
+
+import { checkAccess } from '@mds-core/mds-api-server'
+
+function api(app: express.Express): express.Express {
+  app.get(
+    pathsFor('/geographies/meta/'),
+    checkAccess(scopes => scopes.includes('policies:read')),
```

Comment:
I don't think we actually need _any_ scope for this, and if we did, it shouldn't be a policies scope

Reply:
Why? Policies rely on this. If you can't have any rando modify a policy, why should they be allowed to modify a geography?